apiv2/src/admin/core/sejours/phase1/desistement/ValiderDesisterPostAffectation.ts

```diff
@@ -39,32 +40,64 @@ export class ValiderDesisterPostAffectation implements UseCase<DesisterValiderTa
         const ids = simulationData.jeunesADesister.map((jeune) => jeune.id);
         const jeunes = await this.jeuneGateway.findByIds(ids);
 
-        const { jeunesAutreSession, jeunesConfirmes, jeunesDesistes, jeunesNonConfirmes } =
-            this.desistementService.groupJeunesByReponseAuxAffectations(jeunes, sessionId);
+        const jeunesByStatus = this.desistementService.groupJeunesByReponseAuxAffectations(jeunes, sessionId);
```

Comment:
```suggestion
        const { jeunesNonConfirmes, jeunesConfirmes, jeunesAutreSession,  jeunesDesistes } = this.desistementService.groupJeunesByReponseAuxAffectations(jeunes, sessionId);
```
que penses tu de ça ?
vu la simplicité du retour ça permet de ne plus avoir de `Object.entries(jeunesByStatus).reduce`, que je trouve plus lisible
```
rapportData = {
                jeunesNonConfirmes: mapJeunesTraitement(jeunesNonConfirmes),
                jeunesConfirmes: mapJeunesTraitement(jeunesConfirmes),
                jeunesAutreSession: mapJeunesTraitement(jeunesAutreSession),
                jeunesDesistes: mapJeunesTraitement(jeunesDesistes),
            }
```